Rakefile

```diff
@@ -26,15 +25,25 @@ namespace 'gem' do
       # ~/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/gem_helper.rb:39:in `install'
       def build_gem
         # Ensure that NPM packages are installed before building.
-        sh('yarn install --prod'.shellsplit)
+        sh('yarn install --prod')
   
         default_build_gem
       end
     end
   end
 end
 
-RUBY_VERSIONS=%w[2.5 2.6 2.7]
+RUBY_VERSIONS=%w[2.5 2.6 2.7].select do |version|
```

Comment:
This is the simplest way to get the behavior we want without doing a major build refactor.